make-release.sh

```diff
@@ -30,14 +30,6 @@ bump_version () {
   BUMP_BRANCH=$2
 
   git checkout "${BUMP_BRANCH}"
-
-  echo "Updating project version to ${NEXT_VERSION}"
-  echo "${VERSION}" > VERSION
```

Comment:
Don't we need to update version.go value here instead?

Reply:
yes!